src/Storage/AxolotlStore/TSStorageManager+IdentityKeyStore.m

```diff
@@ -199,6 +199,31 @@ - (nullable OWSRecipientIdentity *)unconfirmedIdentityThatShouldBlockSendingForR
     }
 }
 
+- (nullable OWSRecipientIdentity *)unseenIdentityChangeForRecipientId:(NSString *)recipientId
+{
+    OWSAssert(recipientId != nil);
+
+    @synchronized([[self class] sharedIdentityKeyLock])
+    {
+        OWSRecipientIdentity *currentIdentity = [OWSRecipientIdentity fetchObjectWithUniqueID:recipientId];
```

Comment:
Hmmm... why? Keep in mind that OWSRecipientIdentity already have a similarly constrained (no-cache) dbConnection.